app/models/zoning-map-index.js

```diff
@@ -1,6 +1,19 @@
 import { attr } from '@ember-data/model';
+import { alias } from '@ember/object/computed';
+import { computed } from '@ember/object';
 import CartoGeojsonFeature from './carto-geojson-feature';
 
 export default class ZoningMapIndex extends CartoGeojsonFeature {
   @attr properties;
+
+  @alias('properties.id') title;
+
+  // eslint-disable-next-line ember/require-computed-property-dependencies
+  @computed('title')
```

Comment:
If you do `@computed('title', 'title.length')` I think you can get rid of the eslint-disable on the above line.